onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -85,9 +86,9 @@ BackendManager::BackendManager(const GlobalContext& global_context,
       concrete_backend_ = BackendFactory::MakeBackend(*model_proto_,
                                                       GetGlobalContext(),
                                                       subgraph_context_);
-    } catch (std::string const& msg) {
+    } catch (const OnnxRuntimeException& ex) {
```

Comment:
In ov interface.cc and basic_backend.cc we are throwing ORT_THROW(msg) on getting exception
should we not throw exception